package-configuration/linux-cli/get-cloudify.py

```diff
@@ -122,7 +125,9 @@ def run(cmd, sudo=False):
     # while the process is still running, print output
     while proc.poll() is None:
         if VERBOSE:
-            _print_proc_out(proc)
+            while _print_proc_out(proc):
+                pass
+        time.sleep(1)
     _print_proc_out(proc)
```

Comment:
Generally it looks like a good fix, but:
1. why call print_proc_out until it returns False (no more output) and then out of the look read it again? seems like the seconds call is redundant (would always return false). If it would make sense only in a non-verbose mode, let's put it in an "else" (if verbose.... else)
2. I think it would be even better if you move the while loop into _print_proc_out